toonz/sources/toonz/xsheetviewer.cpp

```diff
@@ -694,18 +835,12 @@ void XsheetViewer::paintEvent(QPaintEvent*)
 //-----------------------------------------------------------------------------
 
 void XsheetViewer::resizeEvent(QResizeEvent *event) {
-  int w              = width();
-  int h              = height();
-  int scrollBarWidth = 16;
-  m_noteScrollArea->setGeometry(3, 1, m_x0 - 4, m_y0 - 3);
-  m_cellScrollArea->setGeometry(m_x0, m_y0, w - m_x0, h - m_y0);
-  m_columnScrollArea->setGeometry(m_x0, 1, w - m_x0 - scrollBarWidth, m_y0 - 3);
```

Comment:
As you can see here, there were 2 pixel spacing between the column area and the cell area in the previous version. IMO it would better to retain such spacing in order to clarify the appearance.